objects/src/accounts/seed.rs

```diff
@@ -137,10 +137,10 @@ pub fn get_account_seed_single(
     let init_seed: Vec<[u8; 8]> =
         init_seed.chunks(8).map(|chunk| chunk.try_into().unwrap()).collect();
     let mut current_seed: Word = [
-        Felt::from(init_seed[0]),
-        Felt::from(init_seed[1]),
-        Felt::from(init_seed[2]),
-        Felt::from(init_seed[3]),
+        Felt::try_from(init_seed[0]).expect("value is greater than or equal to the field modulus"),
+        Felt::try_from(init_seed[1]).expect("value is greater than or equal to the field modulus"),
+        Felt::try_from(init_seed[2]).expect("value is greater than or equal to the field modulus"),
+        Felt::try_from(init_seed[3]).expect("value is greater than or equal to the field modulus"),
     ];
```

Comment:
I think `Felt::new()` is probably fine here. Or if not, we should return an error rather than panicking.

Reply:
Yes, I'd add `InvalidInitSeed` error variant to `AccountError`.